src/drafter.cc

```diff
@@ -45,20 +45,16 @@ DRAFTER_API drafter_error drafter_parse_blueprint_to(const char* source,
         return DRAFTER_EINVALID_INPUT;
     }
 
-    if (!out) {
-        return DRAFTER_EINVALID_OUTPUT;
-    }
-
     drafter_result* result = nullptr;
-    *out = nullptr;
 
     drafter_error ret = drafter_parse_blueprint(source, &result, parse_opts);
 
     if (!result) {
         return ret;
     }
 
-    *out = drafter_serialize(result, serialize_opts);
+    if(out)
```

Comment:
All the rest of the lines use the block form of `if(out) { ... }`. What's the coding style rules on this?

Reply:
We don't have a rule on this; I tend to omit the brackets based on look & taste (TM).
If you think the missing brackets are misleading here, I should add them :) .

Reply:
I'd vote to be consistent at least in a single file. So yes, please :)

Reply:
It's good you pointed this out, judging by the missing space after the `if` keyword and green CI earlier, our formatting CI test must be broken. Good to know, thanks for the insight from outside ;-)